### PR TITLE
Transition from Helm Repositories to OCI Artifacts for Storing Helm Charts - update to helm3 command

### PR DESCRIPTION
**Pull request by NickAzureDevops**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
update helm to using helm 3 command . https://tools.hmcts.net/jira/browse/DTSPO-19019

### CODE REVIEW

File: src/uk/gov/hmcts/contino/Helm.groovy [hunk 1]
```diff
@@ -37,6 +37,7 @@ class Helm {
   def setup() {
     configureAcr()
     removeRepo()
+    authenticateAcr()
```
[the remark this answers is not in the document]
Comment (NickAzureDevops) [reply src/uk/gov/hmcts/contino/Helm.groovy:40]: what you mean by called first?

[the remark this answers is not in the document]
Comment (reespozzi) [reply src/uk/gov/hmcts/contino/Helm.groovy:40]: Before we configure acr / remove repo

Will those funcs be updated in the future too or?

[the remark this answers is not in the document]
Comment (NickAzureDevops) [reply src/uk/gov/hmcts/contino/Helm.groovy]: done

**Comment by NickAzureDevops** [comment]
Tested here https://github.com/hmcts/sds-toffee-frontend/pull/544

**Review by MartyFox — APPROVED** [review]

**Pull request merged**